ecosystem_tests/ecosystem_tests_cli/__init__.py

```diff
@@ -0,0 +1,14 @@
+########
+# Copyright (c) 2014-2019 Cloudify Platform Ltd. All rights reserved
```

Comment:
We should update all copyrights to 2014 - 2021

Reply:
Done.